omero/new/index.html

```diff
@@ -10,11 +10,17 @@
         <h2 class="header-blue">NEW Features</h2>
         <hr class="medium-10">
         <p class="subheader">
-            The latest major release of OMERO includes the following new functionality
+            The most recent releases of OMERO include the following new functionality
         </p>
         <hr class="medium-10">
         <div class="row small-12 medium-up-3">
             <div class="omero-feature-container column text-center">
+                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_restricted-admins.svg">
+                <h4>Restricted Admins</h4>
+                <p>New configurable user role allowing trusted users to act on behalf of others.</p>
+                <a class="button hollow tiny" href="https://docs.openmicroscopy.org/latest/omero/sysadmins/admins-with-restricted-privileges.html" target="_blank">View administrator guide</a>
```

Comment:
This URL doesn't work.  Is that intentional?

Reply:
Looks good as https://docs.openmicroscopy.org/omero/5.4.0-m3/sysadmins/admins-with-restricted-privileges.html.

Reply:
Yep, as mentioned in a comment above, it'll work now with an milestone version number, the latest redirect will be live when we release.